version/rawversion/version.go

```diff
@@ -7,4 +7,4 @@ package rawversion
 //
 // NOTE: remember to bump the version at the top of the top-level README.md
 // file when this is bumped.
-const RawVersion = "5.3.0"
+const RawVersion = "5.3.1-dev"
```

Comment:
We generally don't do "-dev" in release branches, I'd remove this commit.

Reply:
We always did do -dev in podman upstream branches so I am fine with this. Although I would like to see follow a more consistent pattern across our repos (especially around branch names)